packages/tokamak/sdk/examples/deposit-withdraw-ton-js/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think you can make it hardhat task!
You can refer to https://hardhat.org/hardhat-runner/docs/advanced/create-task and https://github.com/tokamak-network/tokamak-titan-canyon/blob/04b824af3d7dc6658ab35207866d4db2d4787276/packages/tokamak/sdk/tasks

Reply:
Thank you!
Let me see how to use hardhat's tasks